### PR TITLE
Update if.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
remove extra "have" from line 36

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/spreadsheets/functions-and-formulas/logical-functions/if.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/spreadsheets/functions-and-formulas/logical-functions/if.md?ref=8ab65cdffe707928d8a9766683ff71a9d0368bef)

File: spreadsheets/functions-and-formulas/logical-functions/if.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Example in both cases:
 
 ![if-example](https://img.enkipro.com/5561b64a6cb62ff33b21599c8aacb2d6.png)
 
-What the function outputs when its `TRUE` or `FALSE` is up to you. We could've have put "banana", "abgb315123ogua", or anything else.
+What the function outputs when its `TRUE` or `FALSE` is up to you. We could've put "banana", "abgb315123ogua", or anything else.
```
Comment (kapnobatai137) [inline spreadsheets/functions-and-formulas/logical-functions/if.md:36]:
```suggestion
What the function outputs when it's `TRUE` or `FALSE` is up to you. We could've put "banana", "abgb315123ogua", or anything else.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: spreadsheets/functions-and-formulas/logical-functions/if.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply spreadsheets/functions-and-formulas/logical-functions/if.md:36]: I remember fixing that a couple of days ago. I might've forgotten to push the commit 😅

Comment (kapnobatai137) [reply spreadsheets/functions-and-formulas/logical-functions/if.md]: haha, no worries

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**